_data/gaming.yml

```diff
@@ -369,13 +367,12 @@ websites:
     tfa:
       - sms
       - email
-      - software
+      - totp
     doc: https://support.microsoft.com/en-us/help/12408/
 
   - name: YoYo Games
     url: https://www.yoyogames.com/
     img: yoyogames.png
     tfa:
       - email
-      - software
-    doc: https://help.yoyogames.com/hc/en-us/articles/216757288-Creating-a-YoYo-Account
```

Comment:
I can't find anything on their website that confirms they support 2FA. It might be best just to remove the entry completely for now.
It probably shouldn't even be in this category to begin with.

Reply:
I changed the doc to point to the screenshot embedded in this PR (see below).

The entry may as well fit in the "Developer" category, but I think it's more raleted to Gaming.